src/pages/projectSettings/Tabs.tsx

```diff
@@ -120,6 +120,10 @@ export const ProjectSettingsTabs: React.FC<Props> = ({
             repoData={
               tabData[ProjectSettingsTabRoutes.GithubCommitQueue].repoData
             }
+            versionControlEnabled={
+              projectData?.projectRef?.versionControlEnabled ??
```

Comment:
For a project, `versionControlEnabled` may be false. If so, we do not want to use the repo value—we should continue to use the project value.

This would be functionally the same if you think it's cleaner:
```js
versionControlEnabled={
  projectType === ProjectType.Repo
    ? repoData.projectRef.versionControlEnabled
    : projectData.projectRef.versionControlEnabled
}
```

Reply:
That makes sense. I was just double checking that it was intentional as opposed to just being used interchangeably.